src/utils/calculateDDay.ts

```diff
@@ -0,0 +1,29 @@
+import { differenceInDays, parse } from "date-fns";
+
+const calculateDDay = (deadline: string): number | never => {
+  // Validate date format
+  const parsedDeadline = parse(deadline, "yyyy.MM.dd", new Date());
```

Comment:
실제로 API 데이터는 ISO 포맷으로 들어오기 때문에 나중에 고려를 해야합니다

Reply:
```suggestion
  const parsedDeadline = new Date(deadline);
```
그냥 이렇게 하면 알아서 Date로 바꿔줄 거 같아요!

Reply:
고렇네요 역시 자스 짱이다

Reply:
> 

안됩니다...ㅠㅠ

Reply:
@dohyun-ko 저거 안됩니다 ㅠㅠ

Reply:
2023-08-03T11:57:18.740Z 요렇게 들어옵니당

Reply:
new Date로 yyyy.MM.dd 형식은 파싱이 되던데 왜 안 되는거지,,?